src/internal/plugins/drawers-controller.ts

```diff
@@ -1,13 +1,11 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import { DrawerItem } from '../../app-layout/drawer/interfaces';
-import { IconProps } from '../../icon/interfaces';
 
-export type DrawerConfig = Omit<DrawerItem, 'content'> & {
+export type DrawerConfig = Omit<DrawerItem, 'content' | 'trigger'> & {
```

Comment:
is it still required to exist in `DrawerItem` interface?

Reply:
Yes, because the normal React props API still supports this feature